scripts/us_cdc/500_places/parse_cdc_places.py

```diff
@@ -0,0 +1,349 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+Author: Padma Gundapaneni @padma-g
+Date: 8/19/2021
+Description: This script cleans a csv file of census tract, county,
+or city level CDC 500 PLACES data downloaded from the CDC.
+URL: https://chronicdata.cdc.gov/browse?category=500+Cities+%26+Places
+@input_file     filepath to the original csv that needs to be cleaned
+@output_file    filepath to the csv to which the cleaned data is written
+python3 parse_cdc_places.py input_file output_file
+'''
+
+import sys
+import pandas as pd
+import numpy as np
+
+# Mapping of measure abbreviations to StatVar dcids
+MEASURE_TO_STATVAR_MAP = {
```

Comment:
Which of these measures are new and missing from cdc500?

Reply:
Percent_Person_21To65Years_Female_ReceivedCervicalCancerScreening and Percent_Person_65OrMoreYears_WithAllTeethLoss are the only new ones. The others already exist.